docs/templating/markdown.html

```diff
@@ -104,32 +104,63 @@ <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
+        <p>Now we can register the KituraMarkdown Template Engine to our router instance.</p>
         <p>Inside the <strong>postInit()</strong> function add: </p>
         <pre><code class="language-swift">router.add(templateEngine: KituraMarkdown())</code></pre>
-        <p>This tells the server that on this instance of the router we'd like to use the KituraMarkdown template engine.</p>
         <div class="info">
           <p>Kitura supports multiple template engines being registered to a single instance of a router. By default each templating engine will handle files in the <strong>./Views</strong> directory that match the file extension it supports.</p>
         </div>
-        <p>We've just added the KituraMarkdown template engine to our router, so now we can use it in a route.</p>
-        <p>Just below our other registered routes we can create a new route for KituraMarkdown:</p>
-        <pre><code class="language-swift">router.get("/book") { request, response, next in
-    try response.render("book.md", context: [:])
-}</code></pre>
-        <p>The <strong>response.render()</strong> method attempts to populate the provided template file with the provided data.</p>
-        <p>However, as we are rendering a Markdown file the <strong>context</strong> parameter is ignored.</p>
-        <p>So in our case KituraMarkdown will take the <strong>book.md</strong> file and render the Markdown in it to be viewed in a browser.</p>
-        <p>This obviously will not work as we haven't created our <strong>book.md</strong> file.</p>
         <div class="underline"></div>
-        <h2 class="heading-2"><span class="blue-text bold">Step 2:</span> Create the template file</h2>
+        <h2 class="heading-2"><span class="blue-text">Step 2:</span> Configure KituraMarkdown</h2>
+        <p>By default KituraMarkdown will render our Markdown to their corresponding HTML elements without allowing us to set any HTML properties, such as the configuration found in the <strong>&lt;head&gt;</strong> tag.</p>
+        <p>To provide this configuration we can provide a HTML wrapper template for our rendered Markdown using MarkdownOptions.</p>
+        <p>Inside the <strong>postInit()</strong> function add:</p>
+        <pre><code class="language-swift">let markdownOptions = MarkdownOptions(pageTemplate: "default")</code></pre>
+        <div class=info>
+            <p>We have selected the default template which provides the following HTML wrapper:</p>
+            <pre><code>&lt;!DOCTYPE html&gt;
+    &lt;html lang=\"en\"&gt;
+      &lt;head&gt;
+        &lt;meta charset=\"UTF-8\"&gt;
+      &lt;/head&gt;
+      &lt;body&gt;
+        &lt;div&gt;
+          &lt;snippetInsertLocation&gt;&lt;/snippetInsertLocation&gt;
+        &lt;/div&gt;
+      &lt;/body&gt;
+    &lt;/html&gt;
+            </code></pre>
+            <p>Alternatively you can provide your own template using the MarkdownOptions:</p>
```

Comment:
Can we remove 'the' so it reads:
Alternatively you can provide your own template using MarkdownOptions: